sdk/python/kfp/compiler/compiler.py

```diff
@@ -219,16 +219,12 @@ def _process_args(self, raw_args, argument_inputs):
     processed_args = list(map(str, raw_args))
     for i, _ in enumerate(processed_args):
       # unsanitized_argument_inputs stores a dict: string of sanitized param -> string of unsanitized param
-      matches = []
-      matches += _match_serialized_pipelineparam(str(processed_args[i]))
+      param_tuples = []
+      param_tuples += _match_serialized_pipelineparam(str(processed_args[i]))
```

Comment:
Thanks for the fix. The code looks much better now.

Reply:
Thank you for pointing that out. I think it is important to address these concerns to keep the maintenance cost low.